### DISCUSSION TITLE
CLI Stopped Working

### DISCUSSION THREAD

**Discussion by hodgesz** [microsoft/graphrag#1327] [Q&A]
Any reason why the CLI would stop working on Mac? The index and query functions use to work fine from the CLI but now:

graphrag index --root ./ragtest
zsh: command not found: graphrag

I have tried creating a new Conda environment and fresh pip install graphrag with the same result. I have tried this on 2 separate Mac laptops and still can't figure out what is going on.

Any ideas?

**Comment by owquresh** [comment]
It doesn't work for me either and I have tried on mac and ubuntu. Was trying to debug it for an entire day because I thought my python was messed up.

**Comment by paolotamag** [comment]
me three, can we see if any updates were pushed to pip? and if not, how can it stop working just like that?
I followed this [tutorial](https://microsoft.github.io/graphrag/get_started/) and after a few hours it gave me the 'command not found', i tried on a new virtual or conda env, same thing! 
Also tried on Windows, same thing, is the CLI gone?

**Comment by godwin05** [comment]
Give this command : python -m graphrag.index --init --root ./ragtest

**Reply by owquresh** [reply]
This command works for me

**Comment by paolotamag** [comment]
it seems related to this commit?
https://github.com/microsoft/graphrag/commit/083de12bcf9e68e51032500106dc4aff771445a4

either way I am still a bit stuck.

this worked:

`python -m graphrag.index --init --root ./ragtest`

instead of this:

`graphrag init --root ./ragtest`

but then this did not:

`python -m graphrag.index --root ./ragtest`

instead of

`graphrag index --root ./ragtest`

and i am getting

```
FileNotFoundError: [Errno 2] No such file or directory: 
'/Users/path/to/folder/ragtest/prompts/entity_extraction.txt'
```

does anybody know the command that works now to execute the pipeline?

**Reply by paolotamag** [reply]
fixed: the above works, just make sure to have all the files created by the initialization (i was missing the `prompts` folder by mistake)
